cmake/OTelMakeRpm.cmake

```diff
@@ -0,0 +1,23 @@
+set(CMAKE_PROJECT_NAME "opentelemetry-cpp")
+set(CPACK_SYSTEM_NAME "${OS_RELEASE_ID}-${OS_RELEASE_VERSION_ID}")
+set(CPACK_PACKAGING_INSTALL_PREFIX "/usr/local")
+set(CPACK_GENERATOR "RPM")
+
+set(CPACK_RPM_PACKAGE_DESCRIPTION "OpenTelemetry C++ for Linux")
+set(CPACK_RPM_PACKAGE_DESCRIPTION_SUMMARY "OpenTelemetry C++ for Linux - C++ Implementation of OpenTelemetry Specification")
+set(CPACK_RPM_PACKAGE_CONTACT "OpenTelemetry C++")
+
+set(MAJOR_VERSION "1")
+set(MINOR_VERSION "6")
+set(MINOR_VERSION "1")
+
+set(CPACK_PACKAGE_DESCRIPTION "OpenTelemetry C++ for Linux")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "OpenTelemetry C++ for Linux - C++ Implementation of OpenTelemetry Specification")
+set(CPACK_PACKAGE_VENDOR "OpenTelemetry")
+set(CPACK_PACKAGE_CONTACT "OpenTelemetry C++")
+set(CPACK_PACKAGE_VERSION_MAJOR "${MAJOR_VERSION}")
```

Comment:
Why not use `${PROJECT_VERSION_MAJOR}` , `${PROJECT_VERSION_MINOR}` and `${PROJECT_VERSION_PATCH}` ?

Reply:
Good point. We don't set these variables in our cmake, but have changed code to use `OPENTELEMETRY_VERSION`.